.github/workflows/publish.yml

```diff
@@ -30,6 +39,12 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v4
 
+      - name: Delete cache
+        # When pushing to main, clear the cache.
```

Comment:
What is the effect of clearing the transcripts cache here? Maybe extend the comment with that information.

Reply:
Changed the comment.
done.